src/debugger.ts

```diff
@@ -146,6 +151,7 @@ export namespace Debugger {
     private _mode: IDebugger.Mode;
     private _modeChanged = new Signal<this, IDebugger.Mode>(this);
     private _currentLineChanged = new Signal<this, number>(this);
+    private _clearLines = new Signal<this, void>(this);
```

Comment:
It would be great to keep the same naming convention for all the signals:

```suggestion
    private _linesCleared = new Signal<this, void>(this);
```